src/components/PresentBar/PresentBar.tsx

```diff
@@ -43,6 +86,24 @@ export function PresentBar() {
           checkedIcon={<PresentToAllIcon />}
         />
       )}
+      {isViewingPresentation && (
```

Comment:
The name of the variable is wrong because it is only displayed for the presenter not for the ones that view the presentation. But this file uses the pattern to use the state.type variable directly so we can change it to:

```suggestion
      {state.type === 'presenting' && (
```

and skip the variable.